cluster/addons/dns/README.md

```diff
@@ -89,8 +89,8 @@ of this yourself.  First, each kubelet needs to run with the following flags
 set:
 
 ```
---cluster_dns=<DNS service ip>
---cluster_domain=<default local domain>
+--cluster-dns=<DNS service ip>
```

Comment:
The risk of this PR is best captured by the very first hunk of diff - you're changing the docs for kube2sky config, but not changing the flag, and even if you had caught those flags it would not be correct because kube2sky doesn't use pflags (yet), so it would be an incompatible change.

I think this process needs to be more nuanced and less bulk.  For each cmd/* that you can prove uses pflags SetNormalizeFunc(), for each flag therein, global search-replace foo_bar with foo-bar.

Then we have a clearer picture of whats left...